### PR TITLE
Service READMEs

### PR DESCRIPTION
**Pull request by nmonterroso**
https://wikia-inc.atlassian.net/browse/SERVICES-104
@drsnyder @pchojnacki @sqreek 

Adding base README to new service template, as well as a `README` to existing services that I'm familiar with.

### CODE REVIEW

File: service/clickstream/README.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
-# Pandora-example
```
Comment (nmonterroso) [inline service/clickstream/README.md]: @pchojnacki - can you make sure this is accurate, and add any info that's missing?

File: service/discussion/README.md [hunk 2]
```diff
@@ -40,6 +40,12 @@ This project uses [lombok](http://projectlombok.org) and [Guice](https://github.
 To get this to work with IntelliJ you must enable
 [Annotation Processing](https://www.jetbrains.com/idea/help/configuring-annotation-processing.html)
 
+# API Docs
```
Comment (nmonterroso) [inline service/discussion/README.md]: @jogura @jimmordino @armonr - can you make sure this is accurate, and add any info that's missing?

File: service/notifications/README.md [hunk 3]
```diff
@@ -1,14 +1,18 @@
 # Notifications-service prototype
```
Comment (nmonterroso) [inline service/notifications/README.md]: @rafalkalinski @kvas-damian - can you make sure this is accurate, and add any info that's missing?

File: service/user-preference/README.md [hunk 4]
```diff
@@ -2,7 +2,13 @@
 
 A service providing access to a white-listed set of user preferences. 
 
-# How to use it?
+# API Docs
+Visit `/swagger` on a running instance of the service to view [Swagger](http://swagger.io/) documentation
```
Comment (nmonterroso) [inline service/user-preference/README.md]: @drsnyder - can you make sure this is accurate, and add any info that's missing?

File: new-service-template/README.md [hunk 5]
```diff
@@ -1,22 +1,25 @@
...
+Please add information about any gotchas or caveats when running this service, such as a required JVM argument or environment variable that is not one of the standard environment variables.
 
-To make a request from the server:
+# Gradle Tasks
+
+To run the server:
 
 ```bash
-curl -v http://localhost:8080/HelloWorld/Your_Name_Here
+../../gradlew run # launch application
```
Comment (sqreek) [inline new-service-template/README.md]: it will be nice to add from with folder it should be run

File: service/notifications/README.md (hunk 3 above)
Comment (kvas-damian) [inline service/notifications/README.md]: done!

File: service/discussion/README.md (hunk 2 above)
Comment (jogura) [inline service/discussion/README.md]: Thank you. Updated here: 13711a8

**Pull request merged**